src/components/common/Balance.js

```diff
@@ -12,9 +12,9 @@ const Balance = ({ amount, milli }) => {
     if (!milli) {
         throw new Error("token image should not be null")
     }
+    amount = amount.toString()
```

Comment:
this shouldn't be needed, If there is any more cases when `amount` is number, we need to find and fix the root cause, e.g. like here:
https://github.com/nearprotocol/near-wallet/commit/ec6c1c687e73b587b34c6264f3c9e3170a8be16c#diff-f60c724c7de0318f2d59895aadc93156L112